pvr.nextpvr/resources/settings.xml

```diff
@@ -1,25 +1,272 @@
+					<default>64</default>
+					<constraints>
+						<minimum>16</minimum>
+						<step>16</step>
+						<maximum>96</maximum>
+					</constraints>
+					<control format="integer" type="slider">
+						<popup>false</popup>
+					</control>
+				</setting>
+				<setting help="" id="chunkrecording" label="30168" type="integer">
+					<level>3</level>
+					<default>32</default>
+					<constraints>
+						<minimum>16</minimum>
+						<step>16</step>
+						<maximum>96</maximum>
+					</constraints>
+					<control format="integer" type="slider">
+						<popup>false</popup>
+					</control>
+				</setting>
+			</group>
+		</category>
+		<category help="" id="advanced5" label="30175">
+			<group id="12">
+				<setting help="30670" id="reseticons" label="30170" type="boolean">
+					<level>2</level>
+					<default>false</default>
+					<control type="toggle"/>
+					<dependencies>
+						<dependency type="visible">
+							<condition operator="is" setting="legacy">false</condition>
+						</dependency>
+					</dependencies>
+				</setting>
+				<setting help="" id="remoteaccess" label="30179" type="boolean">
+					<level>2</level>
+					<default>false</default>
+					<control type="toggle"/>
+					<dependencies>
+						<dependency type="visible">
+							<condition operator="is" setting="legacy">false</condition>
+							<condition operator="!is" setting="pin">0000</condition>
+						</dependency>
+					</dependencies>
+				</setting>
+				<setting help="30504" id="guideartwork" label="30004" type="boolean">
+					<level>1</level>
+					<default>false</default>
+					<control type="toggle"/>
+					<dependencies>
+						<dependency type="visible">
+							<condition operator="is" setting="legacy">false</condition>
+							<condition operator="is" setting="remoteaccess">false</condition>
+						</dependency>
+					</dependencies>
+				</setting>
+				<setting help="30504" id="guideartworkportrait" label="30181" type="boolean" parent="guideartwork">
+					<level>1</level>
+					<default>false</default>
+					<control type="toggle"/>
+					<dependencies>
+						<dependency type="visible">
+							<condition operator="is" setting="guideartwork">true</condition>
+						</dependency>
+					</dependencies>
+				</setting>
+				<setting help="30569" id="kodilook" label="30169" type="boolean">
+					<level>1</level>
+					<default>false</default>
+					<control type="toggle"/>
+				</setting>
+				<setting help="30680" id="flattenrecording" label="30180" type="boolean">
+					<level>2</level>
+					<default>false</default>
+					<control type="toggle"/>
+				</setting>
+				<setting help="30678" id="recordingsize" label="30178" type="boolean">
+					<level>3</level>
+					<default>false</default>
+					<control type="toggle"/>
+					<dependencies>
+						<dependency type="visible">
+							<condition operator="is" setting="legacy">false</condition>
+						</dependency>
+					</dependencies>
+				</setting>
+			</group>
+		</category>
+	</section>
```

Comment:
Thanks a lot for update them, only can you remove the tabs and use only spaces for it :thinking:

It looks on my Linux with tabs then e.g. here:
![Bildschirmfoto vom 2020-04-15 04-58-36](https://user-images.githubusercontent.com/6879739/79294186-dbdc4080-7ed5-11ea-8912-6e00b1cc741c.png)

And text copy produce `<------><------><------><------><------><level>1</level>`.

Therefore only please, but not as a request from me :smile:.

About the release of this addon I can combine your request with https://github.com/kodi-pvr/pvr.nextpvr/pull/119

Reply:
I have a few commits coming for this PR so don't bother combining it as I am making a feature release.  Thanks for the offer.   Sorry about the tabs I do have VS Code setup properly in Ubuntu but I bet editing it in Windows today changed things.

Reply:
I found out how this happened VS Code doesn't modify a file when you change tab characteristics, it is display so saving don't save.  You have to run a convert and then save.  Thanks again

Reply:
One of the things I did a while back with some other addons was run clang format, fixed up and committed folder by folder.

It’s not perfect so you need to check the changes. I also increased the line length from kodi’s default. It was time consuming but worth it in the end as the whole codebase was consistent. Also made reviewing much easier. Just a thought.

Reply:
The documentation says not to do this for existing files but I will consider it as a standalone PR.  I prefer doing this when I somehow break the huge pvrclient-nextpvr.cpp into sub cpp modules similar to how is was done here https://github.com/kodi-pvr/pvr.nextpvr/pull/89/commits/7e42c9a7b7a2c8800edd110e840ce0bc496c5b47 and I don't want to include that with a feature release.

Does .clang-format apply to .xml and .po as well?

Reply:
For kodi the documentation says that. Addons do not have to obey those rules. Generally it’s one dev maintaining each so up to them. Fine to leave to a refactoring PR.

Clang-format only applies to C/C++/Objective C.